packages/components/src/styles/flag/flag.declaration.ts

```diff
@@ -0,0 +1,15 @@
+import type { Style } from '../../declaration';
+
+export default {
+  styleName: 'sd-flag',
+  summary: 'Generates basic styles for flag elements.',
+  status: 'experimental',
+  since: '1.33.2',
```

Comment:
pls adopt right before merging. Pls be aware, that we usually dont have any patch versions => only minor version will be changed.

Reply:
Yes, it's a feature, so should become 1.34.0